source/Halibut.CertificateGenerator/CertificateGenerator.cs

```diff
@@ -78,7 +78,7 @@ static X509Certificate2 AttemptToGenerate(string fullName)
                    };
         }
 
-#if !__MonoCS__
+#if !__MonoCS__ && !NETCOREAPP
```

Comment:
I wonder if we need `__MonoCS__` ?

Reply:
I'm not sure of the original intent, seemed safer to leave it alone